nextflow_schema.json

```diff
@@ -108,6 +108,9 @@
                     "pattern": "^\\S+$",
                     "format": "directory-path",
                     "hidden": true
+                },
+                "allele_scheme": {
+                    "type": "string"
```

Comment:
This this is meant to point to a directory of files, could you add `description`, `pattern`, and `format` to the JSON schema entry here:

This would match for example how kraken2 is configured:

https://github.com/phac-nml/mikrokondo/blob/f0f55a418f7120ab7de4178cfca1e8348f2ea19c/nextflow_schema.json#L101-L103

Reply:
Add three requirements in:  c8e0c8a003520471478e446d9fef5107084608ea